packages/orchestrator-ui-components/src/components/WfoSummary/WfoSummaryCardList.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from 'react';
+
+import { useRouter } from 'next/router';
+
+import {
+    EuiButton,
+    EuiFlexItem,
+    EuiHorizontalRule,
+    EuiPanel,
+    EuiSpacer,
+    useEuiScrollBar,
+} from '@elastic/eui';
+
+import { useOrchestratorTheme } from '@/hooks';
+
+import {
+    SummaryCardListItem,
+    WfoSummaryCardListItem,
+} from './WfoSummaryCardListItem';
+
+export type SummaryCardButtonConfig = {
+    name: string;
+    url: string;
+};
+
+export type WfoSummaryCardListProps = {
+    title: string;
+    items: SummaryCardListItem[];
+    button?: SummaryCardButtonConfig;
+};
+
+export const WfoSummaryCardList: FC<WfoSummaryCardListProps> = ({
+    title,
+    items,
+    button,
+}) => {
+    const router = useRouter();
+    const { theme } = useOrchestratorTheme();
+
+    return (
+        <EuiFlexItem style={{ minWidth: 300 }}>
+            <EuiPanel
+                css={{
```

Comment:
Why not use the styles.ts patttern here?

Reply:
done